src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -29,60 +27,69 @@ object Run  {
     rpr.setLogging(logging)
 
     val id = pipeline.genomicsService.pipelines().run(rpr).execute().getName
-    Log.info(s"$tag ID is $id")
-    new Run(id, pipeline)
+    Log.info(s"$tag JES ID is $id")
+    new Run(id, pipeline, tag)
   }
 
   implicit class RunJesParameters(val params: Seq[JesParameter]) extends AnyVal {
     def toRunMap = params.map(p => p.name -> p.gcs).toMap.asJava
   }
 
+  implicit class RunOperationExtension(val operation: Operation) extends AnyVal {
+    def hasStarted = operation.getMetadata.asScala.get("startTime") isDefined
+  }
+
   sealed trait RunStatus
   trait TerminalRunStatus extends RunStatus
-  final case class Initializing(created: DateTime) extends RunStatus
-  final case class Running(created: DateTime, started: DateTime) extends RunStatus
-  final case class Success(created: DateTime, started: DateTime, finished: DateTime) extends TerminalRunStatus
-  final case class Failed(created: DateTime, started: DateTime, finished: DateTime, errorCode: Int, errorMessage: String) extends TerminalRunStatus
+  case object Initializing extends RunStatus
+  case object Running extends RunStatus
+  case object Success extends TerminalRunStatus
+  final case class Failed(errorCode: Int, errorMessage: String) extends TerminalRunStatus {
```

Comment:
`final` on an implementation of a `sealed trait` feels like belt-and-suspenders, but whatever